src/util/triplea/MapXMLCreator/MapXMLCreator.java

```diff
@@ -0,0 +1,839 @@
+						break;
+					case 13: // showInfoMessage("To change the initial owner of a territory, click on the territory and type in its new owner when it asks for it.", "Help For Current Step");
+						break;
+					case 14: // showInfoMessage("To change the units that a territory begins with, click on the territory and use the window that opens to set the territory's units. To do so, use the + and - buttons below each unit's name to change the quantity of each unit in the territory. You can also edit the text in the result textbox directly to change the territory units. Just type the name of each unit you want to add, followed by ':', followed by the unit quantitiy: Example: 'infantry: 1, artillery: 3, armour: 2, fighter: 1'.", "Help For Current Step");
+						break;
+					case 15:
+						final String[] settings = GameSettingsPanel.s_allSettings.split(", ");
+						Arrays.sort(settings);
+						StringBuilder sb = new StringBuilder();
+						sb.append("<html>Here is a list of all the available game settings:<table border=0><tr><td><table border=0>");
+						int setting_nr = 1;
+						char last_first_letter = 'z';
+						final int third_size_setting = (settings.length + 2) / 3;
+						for (final String setting : settings)
+						{
+							char first_letter = setting.charAt(0);
+							if (first_letter == last_first_letter)
+								sb.append("<tr><td>" + setting + "</td></tr>");
+							else
+								sb.append("<tr><td><big>" + first_letter + "</big>" + setting.substring(1) + "</td></tr>");
+							last_first_letter = first_letter;
+							if (setting_nr % third_size_setting == 0)
+								sb.append("</table></td><td><table border=0>");
+							++setting_nr;
+						}
+						sb.append("</table></td></tr></table></html>");
+						showInfoMessage(
+sb.toString(), "List Of Available Game Settings Choices");
+						break;
+					case 16:
+						showInfoMessage(
+									"The map notes can be entered in the textbox. You can enter plain text or html code. Then just click the 'Save Map To File' button, and save the file in the map's 'games' folder. (Create the 'games' folder if not already created)",
+									"Help For Current Step");
+						break;
+				}
+			}
+		});
+		southRightPanel.add(m_bAvailableChoices);
+	}
+	
+	private void showInfoMessage(final String message, final String title) {
+		JOptionPane.showMessageDialog(null,message,title,JOptionPane.INFORMATION_MESSAGE);
+	}
+	
+	private void setupMainPanel(final JPanel panel)
+	{
+		m_mainPanel.removeAll();
+		m_mainPanel.setLayout(new BorderLayout());
+		panel.setBorder(new MatteBorder(1, 0, 1, 1, Color.DARK_GRAY));
+		m_mainPanel.add(panel,BorderLayout.CENTER);
+		setWidgetActivation();
+	}
+	
+	public void setWidgetActivation()

```

Comment:
Why is this method public?
Can it be named something more descriptive than 'setWidgetActivation?"